recipes/adtk/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "adtk" %}
+{% set version = "0.5.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 4318d18d77f26767ab61f5224c791a876bbc1e9cc34ebf2be92e8a6c86bbb1f3
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+    - pandas >=0.23
+    - numpy >=1.15
+    - matplotlib >=3.0
+    - scikit-learn >=0.20
+    - statsmodels >=0.9
+    - packaging >=17.0
+    - tabulate >=0.8
```

Comment:
Do you actually need most of these to install the package? Or do you just need Python and pip?

Reply:
Thes packages are listed [here](https://github.com/arundo/adtk/blob/develop/setup.cfg)

Reply:
right, but that is runtime deps, not build time. I don't think you need these

Reply:
Done!